install.sh

```diff
@@ -20,41 +25,38 @@ declare -a hooks=(
     "pre-commit"
 )
 
+# providers to be installed
+declare -a providers=(
+    "commons"
+)
+
+# get script directory
+script_directory="$(dirname $0)"
+
 # install each hook
+mkdir -p ".git/hooks"
 for hook in "${hooks[@]}"
 do
     dest=".git/hooks/$hook"
-    if [ -z "$is_local" ]; then
-        url="https://raw.githubusercontent.com/$repo/$branch/hooks/$hook"
-        echo "Download from $url"
-        curl "$url" -o "$dest"
-    else 
-        script_directory="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-        file="$script_directory/hooks/$hook"
-        echo "Copying $file"
-        cp "$file" "$dest"
-    fi
-
+    file="$script_directory/hooks/$hook"
+    echo "Copying hook file: $file"
+    cp "$file" "$dest"
     chmod +x "$dest"
 done
```

Comment:
@pp-ps by doing this, we are forcing user to clone this repo locally and keep it sync. It is actually desirable to use it also as a remote script, as documented [here](https://github.com/pagopa/git-hooks/blob/master/README.md#from-a-remote-script). (that was actually the purpose of the `is_local` flag argument)

Reply:
Yes, I can't say I like blind piping to bash from curl from a security best practice, but if that's a strong requirement feel free to push again against this branch.

Reply:
I think easy adoption is the key here. We pretend `install.sh` to be safe as we own it. Do you see any other exploit other than bad instruction on that file?